ngsi_adapter/src/test/acceptance/README.rst

```diff
@@ -26,3 +26,71 @@ Project Structure
     │   │   └───probe_sample_data
     │   └───settings
     │
+
+
+FIWARE Monitoring Automation Framework
+---------------------------------------
+
+Features:
+
+- Lettuce-Tools support
+- Settings using json files and Lettuce-Tools utility
+- Test report using Lettuce-Tools XUnit output
+- NGSI-Adapter Client
+- Logging
+- Remote NGSI-Adapter log capturing
+- Test data management using templates (resources)
+
+
+Acceptance test execution
+-------------------------
+
+Execute the following command in the test project root directory:
+
+::
+
+  $> cd ngsi_adapter/src/test/acceptance
+  $> lettuce_tools -ft send_data_api_resource -ts comp -sd features/ --tags=-skip -en dev
+
+With this command, you will execute:
+
+- components Test Cases in the 'Development' environment configured in settings/dev-properties.json
+- the send_data_api_resource feature
+- Skipping all Scenarios with tagged with "skip"
+
+
+**Prerequisites**
+
+- Python 2.7 or newer (2.x) (https://www.python.org/downloads/)
+- pip (https://pypi.python.org/pypi/pip)
+- virtualenv (https://pypi.python.org/pypi/virtualenv)
+- Monitoring [NGSI-Adapter] (`Download NGSI-Adapter <https://github.com/telefonicaid/fiware-monitoring>`_)
+
+**Test case execution using virtualenv**
+
+1. Create a virtual environment somewhere *(virtualenv ~/venv)*
+#. Activate the virtual environment *(source ~/venv/bin/activate)*
+#. Go to *ngsi_adapter/src/test/acceptance* folder in the project
```

Comment:
[virtualenv](http://virtualenv.readthedocs.org/en/stable/) says nothing about the name of this variable, but its companion [virtuaenvwrapper](http://virtualenvwrapper.readthedocs.org/en/latest/install.html) uses `WORKON_HOME` exactly for this.